packages/oscal-react-library/src/components/OSCALCatalogManageGroup.tsx

```diff
@@ -0,0 +1,1052 @@
+                        theme.palette.secondary.main,
+                    },
+                  },
+                }}
+              ></TextField>
+            </Grid>
+            <Grid sx={{ height: 23, top: 58, position: "absolute" }} container spacing={2}>
+              <Grid xs={5}></Grid>
+              <Grid xs={7}>
+                <IconButton
+                  size="small"
+                  onClick={handleAddBelow}
+                  sx={{
+                    ":hover": {
+                      backgroundColor: "#023E88",
+                      color: "#FFFFFF",
+                    },
+                  }}
+                >
+                  <InsertIcon />
+                </IconButton>
+                <IconButton
+                  size="small"
+                  onClick={handleDecreaseIndent}
+                  sx={{
+                    ":hover": {
+                      backgroundColor: "#023E88",
+                      color: "#FFFFFF",
+                    },
+                  }}
+                >
+                  <FormatIndentDecreaseIcon />
+                </IconButton>
+                <IconButton
+                  size="small"
+                  onClick={handleIncreaseIndent}
+                  sx={{
+                    ":hover": {
+                      backgroundColor: "#023E88",
+                      color: "#FFFFFF",
+                    },
+                  }}
+                >
+                  <FormatIndentIncreaseIcon />
+                </IconButton>
+                <IconButton
+                  size="small"
+                  sx={{
+                    ":hover": {
+                      backgroundColor: "#ffffff",
+                      color: "#FFFFFF",
+                    },
+                  }}
+                  onClick={handleDeleteGroup}
+                >
+                  <DeleteIcon />
+                </IconButton>
+                <IconButton
+                  size="small"
+                  color="primary"
+                  sx={{
+                    color: "#023E88",
+                    ":hover": {
+                      backgroundColor: "#023E88",
+                      color: "#FFFFFF",
+                    },
+                  }}
+                  onClick={handleSaveNewGroup}
+                >
+                  <SaveIcon></SaveIcon>
+                </IconButton>
+              </Grid>
+            </Grid>
+          </Grid>
+        </Container>
+        <Box
+          sx={{
+            left: 313,
+            width: 6,
+            height: 73,
+            position: "absolute",
+            background: isSelected ? "#FF6600" : "#F6F6F6",
+          }}
+        ></Box>
+      </Container>
+    );
+  };
+  const RootLevel: React.FC = () => {
+    function handleRootLevelClick() {
+      setSelectedItemName("Root");
+    }
+    const rootSelected = selectedItemName === "Root" ? true : false;
```

Comment:
I don't think the ternary operator is needed here. I think `const rootSelected = selectedItemName === "Root"` will return the right value.

Reply:
You are right. Thanks for pointing that out. Thanks!